config/locales/en.yml

```diff
@@ -326,3 +326,11 @@ en:
               min_grade:          "must be greater than 0"
             user:
               blank:              "can't be blank'"
+        assignment:
+          attributes:
+            user:
+              blank:                 "User can't be null"
+            offspring:
+              blank:                 "must exist"
```

Comment:
Exactly, but in the future it will have to be divided into contexts, because when an admin is logged in, 'nil' is a valid value for user. This topic we already discuss it, so for the moment, for clarity I think is better to leave it "not accepting a nil user" and then modify it.